aries_cloudagent/protocols/issue_credential/v2_0/tests/test_manager.py

```diff
@@ -1273,6 +1310,32 @@ async def test_store_credential(self):
             assert ret_cx_rec.cred_issue.attachment() == INDY_CRED
             assert ret_cx_rec.state == V20CredExRecord.STATE_CREDENTIAL_RECEIVED
 
+
+            attachment_id = str(uuid4())
+            data = b"Hello World!"
+            supplements = {
+                "type": "hashlink-data",
+                "ref": attachment_id,
+                "attrs": [{"key": "field", "value": "image"}],
+            }
+            attachments = {
+                "@id": attachment_id,
+                "mime-type": "image/jpeg",
+                "filename": "face.png",
+                "byte_count": len(data),
+                "description": "A picture of a face",
+                "data": {"base64": base64.urlsafe_b64encode(data).decode()},
+            }
+
+            stored_cx_rec.supplements = supplements
+            stored_cx_rec.attachments = attachments
+
+            await self.manager.store_credential(
+                stored_cx_rec, cred_id
+                )
+            mock_handler.return_value.store_supplements.assert_called_once()
```

Comment:
After asserting, should we check that the record was saved with the correct information?